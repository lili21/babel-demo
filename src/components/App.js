import React, { Component } from 'react'
import HOC from './hoc'

@HOC
export default class App extends Component {
  state = { name: 'Babel' }
  render () {
    return (
      <div className="app">
        <h1>Hello, { this.state.name }</h1>
      </div>
    )
  }
}
