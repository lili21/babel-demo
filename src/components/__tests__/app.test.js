import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'

it('renders correctlly', () => {
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
