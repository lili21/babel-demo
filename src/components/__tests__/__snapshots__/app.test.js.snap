// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctlly 1`] = `
<div
  className="app"
>
  <h1>
    Hello React
  </h1>
</div>
`;
